caampr-backend/src/main/java/com/jelistan/caampr/lambda/model/Profile.java

```diff
@@ -1,14 +1,20 @@
 package com.jelistan.caampr.lambda.model;
 
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
 import lombok.Builder;
 import lombok.Data;
 
+@DynamoDBTable(tableName = "Gear-v1")
 @Builder
 @Data
 public class Profile {
+    @DynamoDBHashKey(attributeName = "id")
```

Comment:
The profileId is the id for this table (key), so update `id` to be a string and get rid of the `profileName` field.  Make sense?  Normally, I would have a data model already documented for this, so you're kind of flying blind, so my fault